frontend/components/client/AddCustomerDialog.tsx

```diff
@@ -260,21 +308,16 @@ export const AddCustomerDialog = ({
             <h3>Date of Birth</h3>
           </Box>
 
-          <DatePicker
-            label={formInputs.dob ?? 'Date of birth'}
+          <TextField
+            style={{ marginLeft: '60px' }}
+            required
+            label="YYYY-MM-DD"
+            data-cy="clientdob"
             value={formInputs.dob}
-            data-cy='clientdob'
-            renderInput={(params) => (
-              <TextField
-                {...params}
-                inputProps={{ placeholder: 'YYYY-MM-DD' }}
-                disabled
```

Comment:
As is there is no date format check for input. The only change here should have been removing this line with `disabled` prop.